### DISCUSSION TITLE
connection termination from frontend to backend after 20s

### DISCUSSION THREAD

**Discussion by parshva-shah-dsw** [istio/istio#48945] [Q&A]
### Is this the right place to submit this?

- [X] This is not a security vulnerability or a crashing bug
- [X] This is not a question about how to use Istio

### Bug Description

frontend istio sidecar failing to connect with backend and failing with Upstream error after 20s delay

error logs :- {"kubernetes":{"pod_name":"ApplicationImage","container_image":"applicationimage”:1","namespace_name":"env","pod_id":"123456","annotations":{"sidecar.istio.io/status":"{\"initContainers\":[\"istio-init\"],\"containers\":[\"istio-proxy\"],\"volumes\":[\"workload-socket\",\"credential-socket\",\"workload-certs\",\"istio-envoy\",\"istio-data\",\"istio-podinfo\",\"istio-token\",\"istiod-ca-cert\"],\"imagePullSecrets\":null,\"revision\":\"default\"}"," ":"[{\"eniId\":\"eni-1234\",\"ifAddress\":\"\",\"privateIp\":\"1.0.0.84\",\"ipv6Addr\":\"\",\"vlanId\":12,\"subnetCidr\":\"\",\"subnetV6Cidr\":\"\"}]","kubectl.kubernetes.io/default-logs-container":"app","prometheus.io/path":"/stats/prometheus","kubectl.kubernetes.io/default-container":"app","prometheus.io/port":"15020","istio.io/rev":"default","prometheus.io/scrape":"true"},"labels":{"tags.datadoghq.com/version":"1234","app":"app","service.istio.io/canonical-name":"app","tags.datadoghq.com/env":"app","tags.datadoghq.com/service":"app","admission.datadoghq.com/enabled":"true","version":"b","pod-template-hash":"1234","service.istio.io/canonical-revision":"b","security.istio.io/tlsMode":"istio"},"container_name":"istio-proxy","container_hash":"1234","docker_id":"1234","host":""},"log":"2024-01-21T08:44:43.30982997Z stdout F {\"upstream_cluster\":\"outbound|3198|b|svc.\",\"duration\":20018,\"requested_server_name\":null,\"response_code\":0,\"response_flags\":\"DC\",\"upstream_host\":\"1.0.0.0.\",\"response_code_details\":\"downstream_remote_disconnect\",\"upstream_service_time\":null,\"upstream_transport_failure_reason\":null,\"upstream_local_address\":\"1.0.0.0:60192\",\"start_time\":\"2024-01-21T08:44:23.097Z\",\"user_agent\":\"Apache-HttpClient/4.5.13 (Java/11.0.20.1)\",\"authority\":\"app:3198\",\"request_id\":\xyz\",\"route_name\":null,\"x_forwarded_for\":null,\"downstream_local_address\":\"1.0.0.0:3198\",\"downstream_remote_address\":\"1.0.0.0:34736\",\"protocol\":\"HTTP/1.1\",\"bytes_received\":4830,\"path\":\"/svc/endpoint1 \",\"connection_termination_details\":null,\"bytes_sent\":0,\"method\":\"POST\"}"}

### Version

```Text
istioctl version 1.18.5
k8s version 1.25.3
```

**Comment by zirain** [comment]
duplicated with https://github.com/istio/istio/discussions/48934

**Reply by altdelnow** [reply]
Any resolution to this issue. Recently we observed 5 request failing after waiting for 20 sec wherein usually it takes 7-20 millisecond to clear the transaction. Being a prod environment, it has become critical situation. Any suggestion or pointer will be appreciated.
